### PR TITLE
SCMOD-5674: Exclude copyrights update to sql files

### PR DESCRIPTION
**Pull request by thilagasanthosh**
http://sou-jenkins2.swinfra.net/job/CAFapi/job/CAFapi~caf-common-parent~SCMOD-5674~CI/

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -310,6 +310,8 @@ limitations under the License.
                         <exclude>**/test-data/**</exclude>
                         <exclude>**/test-license/**</exclude>
                         <exclude>**/testcases/**</exclude>
+                        <exclude>**/resources/**/*.sql</exclude>
```
Comment (michael-bryson) [inline pom.xml:313]: Is there a reason you are excluding sql files in only specific locations rather than everywhere?

Comment (thilagasanthosh) [reply pom.xml:313]: @michael-bryson, Now I have updated to exclude all SQL files irrespective of the specific location

Comment (thilagasanthosh) [reply pom.xml:313]: The settings service build passed after this change.

Comment (michael-bryson) [reply pom.xml:313]: @thilagasanthosh I was not telling you to make this change, I was asking was there a reason to only exclude in specific locations? I don't know if excluding everywhere is valid or not, thats why I was asking you. Should we only be excluding sql files that are executed, what about sql files that are not executed but could be in a repository?

Comment (thilagasanthosh) [reply pom.xml:313]: @michael-bryson ,
when we discussed in one standup with @dermot-hardy, he told to exclude only with sql files that causing issues.

**Review by michael-bryson — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -432,6 +434,12 @@ limitations under the License.
                     <plugin>
                         <groupId>com.mycila</groupId>
                         <artifactId>license-maven-plugin</artifactId>
+                        <configuration>
+                            <excludes>
+                                <exclude>**/resources/**/*.sql</exclude>
+                                <exclude>**/docker/**/*.sql</exclude>
+                            </excludes>
+                        </configuration>
```
Comment (dermot-hardy) [inline pom.xml:442]: Surely this doesn't need to be listed twice?  Nothing else had to be listed twice.

Comment (michael-bryson) [reply pom.xml:442]: @thilagasanthosh Have you addressed this comment yet?

Comment (thilagasanthosh) [reply pom.xml:442]: @michael-bryson , As discussed in yesterday's standup based on Dermot comments, the exclusion added to all SQL files. Now the duplicate listing is removed. 
All SQL files exclusion already tested with settings-service and it was working fine.

Comment (michael-bryson) [reply pom.xml:442]: @thilagasanthosh Ok regarding the excluding all sql files.
However the duplicate listing is still there:
- https://github.com/CAFapi/caf-common-parent/blob/develop/pom.xml#L313
- https://github.com/CAFapi/caf-common-parent/blob/develop/pom.xml#L438
Why is there 2 license profiles?

Comment (thilagasanthosh) [reply pom.xml:442]: the change is done in another PR and working fine in settings-service

https://github.com/CAFapi/caf-common-parent/pull/7

**Pull request merged**